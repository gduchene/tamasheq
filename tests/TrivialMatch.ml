let x =
  match 1 < 0 with
  | true  -> "something is terribly wrong here"
  | false -> "correct"
