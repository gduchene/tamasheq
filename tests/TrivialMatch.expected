TrivialMatch_0	=	[1] {
    (no name)	=	correct
}
